### PR TITLE
[ui][new] allow to set webdriver version from maven command to support older browsers

### PR DESCRIPTION
**Pull request by mcada**
//skip-ci

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -59,6 +59,8 @@
 		<el.version>3.0.0</el.version>
 		<box.version>2.30.1</box.version>
 		<jsoup.version>1.11.3</jsoup.version>
+		<chrome.driver.version></chrome.driver.version>
+		<firefox.driver.version></firefox.driver.version>
```
Comment (mcada) [inline pom.xml]: I can delete it from here or leave it empty so it is visible that the property exists from the main pom.xml, what do you suggest @avano ?

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: pom.xml (hunk 1 above)
Comment (avano) [reply pom.xml]: put a comment there that empty == latest

**Comment by avano** [comment]
@mcada what does the version look like? it is just `73` or something else?

File: pom.xml (hunk 1 above)
Comment (asmigala) [reply pom.xml]: I suggest adding it to the readme as well

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**